monochart/values.yaml

```diff
@@ -453,6 +453,7 @@ persistence:
 ingress:
   default:
     enabled: false
+    alb_enabled: false
```

Comment:
instead of making a new variable for this, i think you should test for the className, and unless it is explicitly alb, you can make it disabled.

Reply:
we usually set className as annotation. I'll check if I can extract it easily

Reply:
Code is updated to check ingress class either from the `ingressClass` variable or from the annotation